wagtail/wagtailadmin/templates/wagtailadmin/skeleton.html

```diff
@@ -20,7 +20,12 @@
     <!--[if lt IE 9]>
         <p class="capabilitymessage">{% blocktrans %}You are using an <strong>outdated</strong> browser not supported by this software. Please <a href="http://browsehappy.com/">upgrade your browser</a>.{% endblocktrans %}</p>
     <![endif]-->
-    <noscript class="capabilitymessage">{% trans 'Javascript is required to use Wagtail, but it is currently disabled' %}</noscript>
+    <noscript class="capabilitymessage">
+        {% blocktrans %}
+            For full functionality of this section it is necessary to enable JavaScript.
```

Comment:
I think this first line should still be the same "JS is required to use Wagtail" message. Saying "For full functionality of this section [...]" is confusing at best.